fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Rewriter.java

```diff
@@ -324,7 +325,8 @@ public class Rewriter extends AbstractBatchJobExecutor {
             // this rule should invoke after infer predicate and push down distinct, and before push down limit
             topic("eliminate join according unique or foreign key",
                 bottomUp(new EliminateJoinByFK()),
-                topDown(new EliminateJoinByUnique())
+                topDown(new EliminateJoinByUnique()),
+                topDown(new PullUpJoinFromUnion())
```

Comment:
Is this the right place? Seems after some rules, e.g, group by pushing down pk-fk. We don't have multi-pass rewrite rules running ability right now.

Reply:
Yes, this will be fixed after adding push down agg through join pkfk

Reply:
Suggest to remove this rule's apply until the other one is ready